### PR TITLE
[Localization] Corvus

### PR DESCRIPTION
**Pull request by JavierMatosD**
Localizes messages in the following files: 
* xmlserializer.cpp
* unicode.cpp
* strings.cpp

### CODE REVIEW

File: src/vcpkg/base/unicode.cpp [hunk 1]
```diff
@@ -190,15 +191,15 @@ namespace vcpkg::Unicode
...
-                case utf8_errc::PairedSurrogates:
-                    return "trailing surrogate following leading surrogate (paired surrogates are invalid)";
-                case utf8_errc::UnexpectedContinue: return "found continue code unit in start position";
-                case utf8_errc::UnexpectedStart: return "found start code unit in continue position";
-                case utf8_errc::UnexpectedEof: return "found end of string in middle of code point";
-                default: return "error code out of range";
+                case utf8_errc::NoError: return msg::format(msgNoError).extract_data();
+                case utf8_errc::InvalidCodeUnit: return msg::format(msgInvalidCodeUnit).extract_data();
+                case utf8_errc::InvalidCodePoint:
+                    return msg::format(msgInvalidCodePoint).append_raw(" (>0x10FFFF)").extract_data();
```
Comment (BillyONeal) [inline src/vcpkg/base/unicode.cpp:197]: I don't see any reason to exclude the (>0x10FFFF) bit from the localized message?

File: src/vcpkg/base/unicode.cpp [hunk 2]
```diff
@@ -190,15 +191,15 @@ namespace vcpkg::Unicode
...
+                case utf8_errc::PairedSurrogates: return msg::format(msgPairedSurrogatesAreInvalid).extract_data();
+                case utf8_errc::UnexpectedContinue: return msg::format(msgContinueCodeUnitInStart).extract_data();
+                case utf8_errc::UnexpectedStart: return msg::format(msgStartCodeUnitInContinue).extract_data();
+                case utf8_errc::UnexpectedEof: return msg::format(msgEndOfStringInCodeUnit).extract_data();
+                default: return msg::format(msgErrorCodeOutOfRange).extract_data();
```
Comment (BillyONeal) [inline src/vcpkg/base/unicode.cpp:202]:
```suggestion
                default: Checks::unreachable(VCPKG_LINE_INFO);
```

and delete ErrorCodeOutOfRange

**Review by BillyONeal — APPROVED** [review]
LGTM once these couple of nitpicks are fixed.

File: src/vcpkg/base/unicode.cpp (hunk 1 above)
Comment (JavierMatosD) [reply src/vcpkg/base/unicode.cpp:197]: The message is reused with a different code point

Comment (BillyONeal) [reply src/vcpkg/base/unicode.cpp:197]: Makes sense!

**Review by BillyONeal — APPROVED** [review]

**Pull request merged**